BtConnectorLib/btconnectorlib2/src/main/java/org/thaliproject/p2p/btconnectorlib/BTConnector_BtConnection.java

```diff
@@ -5,217 +5,146 @@
 import android.bluetooth.BluetoothDevice;
 import android.bluetooth.BluetoothSocket;
 import android.content.Context;
-import android.net.wifi.p2p.WifiP2pDevice;
-import android.net.wifi.p2p.WifiP2pManager;
-import android.os.CountDownTimer;
 import android.os.Handler;
 import android.util.Log;
-
-import java.util.Collection;
-import java.util.List;
 import java.util.UUID;
 
 /**
  * Created by juksilve on 13.3.2015.
  */
-public class BTConnector_BtConnection implements BTListenerThread.BtListenCallback, BTConnectToThread.BtConnectToCallback,BTHandShaker.BtHandShakeCallback {
+public class BTConnector_BtConnection implements BTListenerThread.BtListenCallback, BTConnectToThread.BtConnectToCallback{
 
-    BTConnector_BtConnection that = this;
+    private final BTConnector_BtConnection that = this;
 
     public enum State{
-        ConnectionIdle,
-        ConnectionNotInitialized,
         ConnectionConnecting,
         ConnectionConnected
     }
 
     public interface  ListenerCallback{
```

Comment:
Why is this defined at all? Shouldn't it just be the Callback interface from BTConnector?